config/database.yml

```diff
@@ -57,7 +57,7 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: WeEat_test
+  database: WeEat_development
```

Comment:
why are you using the same database for both development and testing? did you mean to commit this?